packages/web-components/src/components/footer/legal-nav.ts

```diff
@@ -23,6 +24,22 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-legal-nav`)
 class DDSLegalNav extends StableSelectorMixin(LitElement) {
+  /**
+   * Size property used for applying classes
+   */
+  @property()
+  size?: string;
```

Comment:
Could you make this an enum? Ref: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/src/components/footer/footer.ts#L48